### ISSUE TITLE
gridfs driver not setting indices

### ISSUE THREAD

**Issue by macrozone** [unchainedshop/unchained#540]
**Describe the bug**

unchained does not set proper indices on the *chunks collections for gridfs buckets, which renders it beeing unusuably slow

normally this is the drivers responsibility to ensure those indices (See https://groups.google.com/g/mongodb-user/c/odh0WKIOD2g)

and i can confirm that other drivers (in my case node  "mongodb": "^3.5.4") set the indizes properly, while unchained does not set them

**To Reproduce**

should be observable with a fresh installation i guess?

**Expected behavior**

unchained sets proper indizes for gridfs

**Comment by pozylon** [comment]
I'll try to reproduce with a fresh installation and if it's observable i'll try to fix it.

**Comment by pozylon** [comment]
I've searched through all of the issues on https://jira.mongodb.org/issues/?jql=summary+%7E+%22gridfs+index*%22+OR+description+%7E+%22gridfs+index*%22+ORDER+BY+lastViewed+DESC 

And i could not find a single issue @macrozone and i honestly don't know how what happens here.

Kitchensink:
- A fresh install of 1.x and upload of a product shows that indexes are there OK
- A fresh install of master and upload of a product shows that indexes are there OK

Legacy Meteor:
- A fresh install of 1.x and upload of a product created the collections but without indexes FAIL


```
const { MongoInternals } = require('meteor/mongo'); // eslint-disable-line
  const db = MongoInternals.defaultRemoteCollectionDriver().mongo.db; // eslint-disable-line
  return db;
```
Versions:
- meteor: METEOR@2.8.0,  mongo@1.16.1 & npm-mongo@4.9.0
- mongodb npm in files: 4.13.0

**Comment by pozylon** [comment]
I have replaced:
```
      const bucket = new mongodb.GridFSBucket(db, options);

```

with 

```
      const bucket = new MongoInternals.NpmModule.GridFSBucket(db, options);
```

MongoDB does some patching through prototyping in their mongodb wrappers, and the unpatched gridfs bucket class somehow does not work properly with the db object.

**Comment by pozylon** [comment]
@macrozone solved, v1.2.53
